wpeverest/user-registration
# Fix - Form setting preview not showing after edit form name

### All Submissions:






### Changes proposed in this Pull Request:

Previously there were some issues which are listed below. This PR will fix all these issues.
1. Form settings preview not showing after editing form name.
2. Not all HTML tags are allowed in any description of any field in the form builder.
3. Login title missing in the Login form.

### How to test the changes in this Pull Request:

1. Navigate to the form builder and edit the form name and check form preview settings showing or not.
2. Drag and drop any field and navigate to field options and check most Html tags are allowed or not
3. Navigate to User Registration settings then navigate to login options of General tab.
4. then check this enable login title option and see login title is visible in the login form or not. 

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form setting preview not showing after edit form name.
